sphinx/api/built-in/rose_arch.rst

```diff
@@ -101,8 +101,8 @@ Example
 Output
 ------
 
-On completion, rose_arch writes a status summary for each target to
-the standard output, which looks like this:
+On completion, :rose:conf:`rose_arch` writes a status summary for each
```

Comment:
Travis has sprung an error because, this should be `` :rose:app:`rose_arch` ``.

Reply:
Oops, will change.